src/main/java/Melchizedek.java

```diff
@@ -0,0 +1,44 @@
+import java.util.Scanner;
+
+public class Melchizedek {
+
+    public static TaskList taskList = new TaskList();
+
+    public static void printHorizontalLine() {
+        System.out.println("\t____________________________________________________________");
+    }
+
+    public static void greetUser() {
+        printHorizontalLine();
+        System.out.println("\tHello! I'm Melchizedek.");
+        System.out.println("\tHow can I be a blessing to you?");
+        printHorizontalLine();
+    }
+
+    public static void sayByeToUser() {
```

Comment:
Good naming convention of methods (), function written in camelCase and is in verb form